update.go

```diff
@@ -298,6 +305,12 @@ other options are ignored.
 			}
 		}
 
+		b := *r.CPU.Burst
+		if config.Cgroups.Resources.CpuPeriod == 0 && b != 0 {
```

Comment:
@Zheaoli Sorry for delayed so much time. Here I understand you mean that `cpu-burst` depends on `cpu-period`? But I think may be we should depends on `cpu-quota`, or maybe we should remove this dependent check? Let it check by the kernel?

Reply:
To be honest, I think it would be great to validate some base rule at the entry of the CLI like https://github.com/opencontainers/runc/blob/main/update.go#L318-L325

If we put the validation process in the kernel, I think the error message will confuse people.

Reply:
> I think the error message will confuse people.

But there is no such check when running a container. There would be different results for ‘run’ and ‘update’ if we only provide ’cpu-burst’.